test-butler-app/src/main/java/com/linkedin/android/testbutler/ButlerService.java

```diff
@@ -56,6 +64,24 @@ public boolean setGsmState(boolean enabled) throws RemoteException {
         public boolean grantPermission(String packageName, String permission) throws RemoteException {
             return permissionGranter.grantPermission(ButlerService.this, packageName, permission);
         }
+
+        @Override
+        public boolean setAccessibilityServiceState(boolean enabled) throws RemoteException {
+            boolean successful = accessibilityServiceEnabler.setAccessibilityServiceEnabled(enabled);
```

Comment:
It may make sense to put the enabler in the ButlerApiStubBase. You'll just have to pass the InstalledAccessibilityServiceProvider to it.

Reply:
IMO, I think it's maybe more confusing to have the logic split between the two classes. ButlerApiStubBase will implement setAccessibilityService, but the code will definitely not work without an override and a waiter. Both of these steps are required for this code to work properly.

Reply:
Yeah, you'd have to override and call super.setAccessibilityServiceState() (emulator) or butlerApi.setAccessibilityServiceState() (physical device). Agreed splitting it probably makes it more confusing though.